### DISCUSSION TITLE
Incompatible version of GaNDLF detected (17)

### DISCUSSION THREAD

**Discussion by xupengfei1009** [mlcommons/GaNDLF#757] [Q&A]
**GaNDLF Version**
0.0.18-dev
-->
Version information of the GaNDLF package in the virtual environment.
gandlf 0.0.17
**Desktop (please complete the following information):**
 - Linux 
 - x86_64-linux-gnu-gcc (Ubuntu 12.2.0-3ubuntu1) 12.2.0

**How did you install GaNDLF**
`conda install -c conda-forge gandlf -y`

**Describe your question/problem**
When I use sample data for model training, the following problem always occurs.The installation process of GANDLF did not prompt any errors.
```
python gandlf_run \
-c 3d_rad_segmentation/sample_training_classification.yaml \
-i train_3d_rad_classification.csv \
 -m /home//GaNDLF/data/ \
-t True \
-d cpu
```
```
The ``converters`` are currently experimental. It may not support operations including (but not limited to) Functions in ``torch.nn.functional`` that involved data dimension
Incompatible version of GaNDLF detected (17)
```

**Comment by sarthakpati** [comment]
Hi @xupengfei1009,

Thank you for your interest in GaNDLF! 

The error you post basically means that there is a mismatch between the config you have and the version of GaNDLF that's installed in your virtual environment. Can you confirm what the `"version"` key in your config file? Also, what is the response of the command (after activating the appropriate virtual environment) `python -c 'import GANDLF as g;print(g.__version__)'`?

**Reply by xupengfei1009** [reply]
python -c 'import GANDLF as g;print(g.__version__)'?
0.0.17

 config file:
 version:
  maximum: 0.0.7
  minimum: 0.0.7

**Reply by xupengfei1009** [reply]
I have already solved the problem, thank you very much for your response
